spec/features/application/application_index_spec.rb

```diff
@@ -30,20 +30,26 @@
                        sex: "male",
                        adopt_status: 'adoptable',
                        shelter_id: @shelter_1.id)
-
-    PetApplication.create(pet_id: @hobbes.id, adopt_application_id: @francis.id)
-    PetApplication.create(pet_id: @hobbes.id, adopt_application_id: @peter_pan.id)
   end
 
   it "shows all applicants on a pet shop page" do
 
+    PetApplication.create(pet_id: @hobbes.id, adopt_application_id: @francis.id)
```

Comment:
good to move these into the other test so we can just test having no connections